camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelEndpointAnnotator.java

```diff
@@ -201,31 +201,7 @@ public String getErrorMessage(EndpointValidationResult result, Map.Entry<String,
     private static class UnknownErrorMsg implements CamelAnnotatorEndpointMessage<String> {
         @Override
         public String getErrorMessage(EndpointValidationResult result, String property) {
-            // for each invalid option build a reason message
-            String returnMsg = "";
-            if (result.getUnknown() != null) {
-                for (String name : result.getUnknown()) {
-                    if (name.equals(property)) {
-                        if (result.getUnknownSuggestions() != null && result.getUnknownSuggestions().containsKey(name)) {
-                            String[] suggestions = result.getUnknownSuggestions().get(name);
-                            if (suggestions != null && suggestions.length > 0) {
-                                String str = Arrays.asList(suggestions).toString();
-                                returnMsg += "\nUnknown option. Did you mean: " + str;
-                            } else {
-                                returnMsg += "\nUnknown option";
-                            }
-                        } else {
-                            returnMsg = "Unknown option";
-                        }
-                    }
-                }
-            }
-            if (result.getRequired() != null) {
-                for (String name : result.getRequired()) {
-                    returnMsg += "\nMissing required option";
-                }
-            }
-            return returnMsg;
+            return "Unknown option";
```

Comment:
We dont have the lucene suggestion enabled. And the code was a bit misleading as you get a call per unknown option.

See screenshot where both 2 unknown options are highlighted
![screen shot 2017-01-10 at 9 50 23 pm](https://cloud.githubusercontent.com/assets/477100/21824282/ed89a18a-d77e-11e6-98c9-26e15af6c8b8.png)